Neos.ContentRepository.Core/Classes/Feature/WorkspaceCommandHandler.php

```diff
@@ -960,13 +960,12 @@ private function requireNonCircularRelationBetweenWorkspaces(Workspace $workspac
         if ($workspace->workspaceName->equals($baseWorkspace->workspaceName)) {
             throw new BaseWorkspaceEqualsWorkspaceException(sprintf('The base workspace of the target must be different from the given workspace "%s".', $workspace->workspaceName->value));
         }
-
         $nextBaseWorkspace = $baseWorkspace;
-        while ($nextBaseWorkspace->baseWorkspaceName !== null) {
+        while (!is_null($nextBaseWorkspace->baseWorkspaceName)) {
```

Comment:
Just wondering: Is that related to the fix? `!is_null(x)` and `x !== null` should behave the same?!

Reply:
Not really, I just wanted to make it consistent with requireBaseWorkspace